libp2p/Host.h

```diff
@@ -240,7 +241,12 @@ class Host: public Worker
     }
 
     /// Get the node information.
-    p2p::NodeInfo nodeInfo() const { return NodeInfo(id(), (networkConfig().publicIPAddress.empty() ? m_tcpPublic.address().to_string() : networkConfig().publicIPAddress), m_tcpPublic.port(), m_clientVersion); }
+    p2p::NodeInfo nodeInfo() const {
+        ENR e;
```

Comment:
You will return empty ENR this way, not useful to anyone.
Get it from `enr()` method

Reply:
Also use clang-format for you changes as described in https://github.com/ethereum/aleth/blob/master/CONTRIBUTING.md

Reply:
Thanks for the reviews!!